house/tests/house_tests.py

```diff
@@ -1,5 +1,10 @@
 import pytest
 from house.models import House, City, Country
+from django.template.loader import get_template
+from django.template import TemplateDoesNotExist
+from house.constants import MINE_EXPENSES_TITLE_ROUTE, MINE_HOUSE_TABLE_ROUTE, MINE_HOUSE_TABLE_TITLE_ROUTE
+from house.constants import MINE_MINE_PAGE_ROUTE, MINE_SIDEBAR_ROUTE
```

Comment:
Same here the const URL, and you could also make only one line of this import

Reply:
Because of the length of the line I split the lines @LiorKGOW